src/test/java/util/xcmailr/util/SendRequest.java

```diff
@@ -48,10 +47,11 @@ public static void login(String email, String password)
 
             br = new BufferedReader(new InputStreamReader(response.getEntity().getContent(), "UTF-8"));
 
-            String output;
-            while ((output = br.readLine()) != null)
+            String output = br.readLine();
```

Comment:
While this does indeed solve the error we had previously, `br` can still be `null` if the `httpClient.execute` method failed.
Hence, you should add a null check to the `br.close` in line 62.

However, why do we only close the `BufferedReader` when an exception is thrown?